src/laser_geometry.cpp

```diff
@@ -302,6 +302,8 @@ const boost::numeric::ublas::matrix<double>& LaserProjection::getUnitVectors_(do
       {
         ranges(i,0) = (double) (scan_in.ranges[i] > range_cutoff
                           ? range_cutoff : scan_in.ranges[i]);
+        ranges(i,0) = (double) (scan_in.ranges[i] < scan_in.range_min
```

Comment:
Values `< range_min`, if we want to convert them from `-INF` to a finite value, they should take a value like `range_min - EPS`, where `EPS` can be something relatively big if we want (it doesn't matter, as long as it helps to distinguish it from a valid `range_min`).